SU-SWS/stanford_basic
# D8CORE-3957: fix the padding right to only appear on the min of lg breakpoint

# READY FOR REVIEW

# Summary
- Fix the event cards when it is a single item in a paragraph row. 
- This piece is to make sure the padding-right only appears on the cards when in  lg breakpoint.

# Review By (Date)
- ASAP

# Urgency
- High

# Steps to Test

1. Pull in this change.
2. And the Events one https://github.com/SU-SWS/stanford_events/pull/63 
2. Clear caches.
3. Create content. Put event cards in one row.
4. Put event cards in another row with something else. 
5. View the page on responsive.
6. Verify the cards are full width with even padding on the sides and the cards have space between.

# Affected Projects or Products
- Events
- SBT, SOE, SAA themes
- Responsive only.
- https://github.com/SU-SWS/stanford_events/pull/63

# Associated Issues and/or People
- D8CORE-3957
- @katrialesser 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
